src/lib/common/RenderFormat.cpp

```diff
@@ -28,49 +28,52 @@
 
 #include "logMsg/logMsg.h"
 #include "logMsg/traceLevels.h"
-#include "common/NotificationFormat.h"
+#include "common/RenderFormat.h"
 #include "common/wsStrip.h"
 
 
 
 /* ****************************************************************************
 *
-* notificationFormatToString - 
+* renderFormatToString - 
 */
-const char* notificationFormatToString(NotificationFormat format, bool noDefault)
+const char* renderFormatToString(RenderFormat format, bool noDefault)
 {
   switch (format)
   {
-  case NGSI_V1_JSON:                return "JSON";
+  case NGSI_V1_LEGACY:              return "legacy";
   case NGSI_V2_NORMALIZED:          return "normalized";
   case NGSI_V2_KEYVALUES:           return "keyValues";
   case NGSI_V2_VALUES:              return "values";
-  case NGSI_NO_NOTIFICATION_FORMAT:
+  case NGSI_V2_UNIQUE_VALUES:       return "uniqueValues";
+  case NO_FORMAT:
     if (noDefault == true)
     {
-      return "no notification format";
+      return "no render format";
     }
     else
     {
       return "JSON";
     }
   }
 
-  return "Unknown notification format";
+  return "Unknown render format";
 }
 
 
 
 /* ****************************************************************************
 *
-* stringToNotificationFormat -
+* stringToRenderFormat -
 */
-NotificationFormat stringToNotificationFormat(const std::string& s, bool noDefault)
+RenderFormat stringToRenderFormat(const std::string& s, bool noDefault)
 {
-  if (s == "JSON")       { return NGSI_V1_JSON;       }
-  if (s == "normalized") { return NGSI_V2_NORMALIZED; }
-  if (s == "keyValues")  { return NGSI_V2_KEYVALUES;  }
-  if (s == "values")     { return NGSI_V2_VALUES;     }
+  if (s == "JSON")         { return NGSI_V1_LEGACY;        }  // Older content in DB
```

Comment:
"Older content in DB" -> "DB content for NGSIv1 rendering due to legacy reasons"

(In the other way, it could be interpreted that before this PR the content in DB for NGSIv1 was "JSON" and now the content in DB for NGSIv1 is "legacy"... which is not corret. In DB "JSON" is still being used for NGSIv1 format).